_includes/tutorials/aggregating-sum/ksql/markup/dev/print-output-topic.adoc

```diff
@@ -0,0 +1,10 @@
+To check that it's working, print out the contents of the output stream's underlying topic:
```

Comment:
One other thing worth mentioning here is how the `AS` keyword functions to make a new column, and thus a new named attribute on the map that's printed. Seems basic, but it looks me a bit to figure that out when I was learning KSQL.

Reply:
Worked that into the previous step.